cmd/up/up.go

```diff
@@ -583,15 +569,32 @@ func (up *upContext) deployApp(ctx context.Context) error {
 		PipelineCMD:        pc,
 		DeployWaiter:       deploy.NewDeployWaiter(k8sClientProvider),
 		EndpointGetter:     deploy.NewEndpointGetter,
+		AnalyticsTracker:   analytics.NewAnalyticsTracker(),
 	}
 
-	return c.RunDeploy(ctx, &deploy.Options{
+	startTime := time.Now()
+	err = c.RunDeploy(ctx, &deploy.Options{
 		Name:             up.Manifest.Name,
 		ManifestPathFlag: up.Options.ManifestPathFlag,
 		ManifestPath:     up.Options.ManifestPath,
 		Timeout:          5 * time.Minute,
 		Build:            false,
 	})
+
+	// tracking deploy either its been successful or not
+	c.AnalyticsTracker.TrackDeploy(analytics.DeployMetadata{
+		Success:                err == nil,
+		IsOktetoRepo:           utils.IsOktetoRepo(),
+		Duration:               time.Since(startTime),
+		PipelineType:           up.Manifest.Type,
+		DeployType:             "automatic",
+		IsPreview:              os.Getenv(model.OktetoCurrentDeployBelongsToPreview) == "true",
+		HasDependenciesSection: up.Manifest.IsV2 && len(up.Manifest.Dependencies) > 0,
```

Comment:
I added couple commits ago a method at manifest to check this prop without conditionals here 

https://github.com/okteto/okteto/blob/master/cmd/exec.go#L104

Maybe we can add one for the `IsRemote` as it depends also from Manifest ?